### PR TITLE
Bump rubygem-hammer_cli_katello to 0.15.0

### PR DESCRIPTION
**Pull request by akofink**
For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [x] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 1]
```diff
@@ -6,12 +6,12 @@
 %global gem_name hammer_cli_katello
 %global confdir hammer
 
-%global release 3
-%global prerelease .pre.master
+%global release 1
+%global prerelease
```
Comment (ehelms) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: It's kinda silly the way it works but you can just comment this out or remove it from the spec file.

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 2]
```diff
@@ -94,6 +94,9 @@ cp -pa .%{gem_dir}/* \
 %doc %{gem_instdir}/test
 
 %changelog
+* Wed Oct 24 2018 Andrew Kofink <ajkofink@gmail.com> - 0.15-0.1
```
Comment (ehelms) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: I think this should be `0.15.0-1`

**Comment by akofink** [comment]
Updated

File: packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec [hunk 3]
```diff
@@ -28,7 +27,7 @@ Obsoletes: rubygem-hammer_cli_gutterball
 
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.15
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.15.0
```
Comment (ehelms) [inline packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: This will break until hammer has bumped to 0.16

Comment (akofink) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: Is this why repoclosure failed?

Comment (ekohl) [reply packages/katello/rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec]: https://github.com/theforeman/foreman-packaging/pull/3133 will be needed I think

**Comment by ekohl** [comment]
[test rpm]

**Pull request merged**